great_expectations/profile/user_configurable_profiler.py

```diff
@@ -1,8 +1,9 @@
 import logging
 import math
+from typing import Callable, Dict, List, Optional, Union, cast
 
-import numpy as np
 from dateutil.parser import parse
+from tqdm.auto import tqdm
```

Comment:
Why doesn't `isort` catch this?

Reply:
@NathanFarmer `PyCharm` was catching it -- hence I moved it.